_data/roles.yml

```diff
@@ -77,6 +77,25 @@
       orcid: 0000-0002-7356-1779
       status: support
       start: "0000-00-00"
+- name: OBO Newsletter Steward
+  open: true
+  commitment: 2-4 hours per month
+  requirements:
+    - Should have a basic journalistic instinct, compiling the most relevant discussion points, decisions and issues for the wider community
+  description: You will be responsible for managing, compiling and disseminating the OBO Newletter
+  responsibilities:
+    - Manage a Google doc with the current and previous newsletter drafts based on the newsletter skeleton
+    - Adding GitHub issues and/or pull requests tagged with "newsletter" to the newsletter, with one or two sentences of summary
+    - Summarising key developments (ongoing and past) to the community, based on reports of the working groups to the Operations Committee meeting
+    - Summarising key decisions from the OBO operations calls
+    - Proactively finalising the newsletter draft for the last meeting of March, June, September and December, and requesting feedback
+    - After 10 days of requesting feedback, send the newsletter to obo-discuss mailing list
+  people:
+    - github: x
```

Comment:
This PR should not be merged before this is filled in.

Reply:
bumping this, what information is missing? Can I track someone down to fill this out?

Reply:
The main info that is missing is the person assigned to this role. I have two volunteers now I will interview next week. Then I will merge it! Thanks for following up.